src/rpc/net.cpp

```diff
@@ -237,13 +237,13 @@ static RPCHelpMan getpeerinfo()
         obj.pushKV("conntime", stats.nTimeConnected);
         obj.pushKV("timeoffset", stats.nTimeOffset);
         if (stats.m_ping_usec > 0) {
-            obj.pushKV("pingtime", ((double)stats.m_ping_usec) / 1e6);
+            obj.pushKV("pingtime", static_cast<double>(stats.m_ping_usec) / 1e6);
         }
         if (stats.m_min_ping_usec < std::numeric_limits<int64_t>::max()) {
-            obj.pushKV("minping", ((double)stats.m_min_ping_usec) / 1e6);
+            obj.pushKV("minping", static_cast<double>(stats.m_min_ping_usec) / 1e6);
         }
         if (stats.m_ping_wait_usec > 0) {
-            obj.pushKV("pingwait", ((double)stats.m_ping_wait_usec) / 1e6);
+            obj.pushKV("pingwait", static_cast<double>(stats.m_ping_wait_usec) / 1e6);
```

Comment:
Still not convinced that it is worth it to change existing code that is being removed in #21015

Reply:
Generally, it might be better to use C++11 non-narrowing casts, or ideally no cast at all.